deb/ubuntu-xenial/Dockerfile.aarch64

```diff
@@ -0,0 +1,36 @@
+FROM aarch64/ubuntu:xenial
+
+RUN apt-get update && apt-get install -y golang-go apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+# Install Go
+# We don't have official binary golang 1.7.5 tarballs for ARM64, eigher for Go or
+# bootstrap, so we use golang-go (1.6) as bootstrap to build Go from source code.
+# We don't use the official ARMv6 released binaries as a GOROOT_BOOTSTRAP, because
+# not all ARM64 platforms support 32-bit mode. 32-bit mode is optional for ARMv8.
+ENV GO_VERSION 1.8.3
+RUN mkdir /usr/src/go && curl -fsSL https://golang.org/dl/go${GO_VERSION}.src.tar.gz | tar -v -C /usr/src/go -xz --strip-components=1 \
+	&& cd /usr/src/go/src \
+	&& GOOS=linux GOARCH=arm64 GOROOT_BOOTSTRAP="$(go env GOROOT)" ./make.bash
+
+ENV GOPATH /go
+ENV PATH /go/bin:/usr/src/go/bin:$PATH
+ENV DOCKER_BUILDTAGS apparmor pkcs11 seccomp selinux
```

Comment:
no need for pkcs11, see https://github.com/moby/moby/commit/b877fc31c51baaf1ec42501ab366dd05a5b6bb21#diff-0a5836d551d7f223bdc8b46372ca0089